be/src/exec/hdfs_scanner_orc.cpp

```diff
@@ -509,15 +528,26 @@ static const std::string kORCProfileSectionPrefix = "ORC";
 void HdfsOrcScanner::do_update_counter(HdfsScanProfile* profile) {
     RuntimeProfile::Counter* delete_build_timer = nullptr;
     RuntimeProfile::Counter* delete_file_per_scan_counter = nullptr;
+    RuntimeProfile::Counter* stripe_sizes_counter = nullptr;
+    RuntimeProfile::Counter* stripe_number_counter = nullptr;
     RuntimeProfile* root = profile->runtime_profile;
 
     ADD_COUNTER(root, kORCProfileSectionPrefix, TUnit::UNIT);
 
     delete_build_timer = ADD_CHILD_TIMER(root, "DeleteBuildTimer", kORCProfileSectionPrefix);
     delete_file_per_scan_counter = ADD_CHILD_COUNTER(root, "DeleteFilesPerScan", TUnit::UNIT, kORCProfileSectionPrefix);
-
     COUNTER_UPDATE(delete_build_timer, _stats.delete_build_ns);
     COUNTER_UPDATE(delete_file_per_scan_counter, _stats.delete_file_per_scan);
+
+    // we expect to get average stripe size instead of sum.
+    stripe_sizes_counter =
+            root->add_child_counter("StripeSizes", TUnit::BYTES,
```

Comment:
`AverageStripeSize`?

Reply:
If it's that, then profile will be like

```
AverageStripeSize
  - MAX_OF_ AverageStripeSize
  - MIN_OF_ AverageStripeSize
```

it makes no sense